src/main/webapp/app/pages/curation/button/RCTButton.tsx

```diff
@@ -13,9 +13,16 @@ import { Button } from 'reactstrap';
 export interface IRCTButtonProps extends StoreProps {
   cancerTypePath: string;
   relevantCancerTypesInfoPath: string; // path to dx, px, or tx
+  readOnly?: boolean;
```

Comment:
I added the readOnly property to RCTButton because I thought this component might be used in multiple places simultaneously. If it’s only used in CurationPage, I’ll remove this line and use StoreProp instead.